doc/source/tutorial/getting_started.rst

```diff
@@ -82,184 +82,137 @@ If ``libboost-dev`` is not already installed on your machine, it may be
 available through your operating system distribution, otherwise it can
 be downloaded from the `libboost <https://www.boost.org/>`_ website.
 
-5. Install ``scons``
+5. Install ``cmake``
```

Comment:
Given that `cmake` is required to install `Charm++`, maybe we should make this step 1.

Reply:
Yes, good point. I'll update this.